src/util/strencodings.cpp

```diff
@@ -59,12 +59,13 @@ signed char HexDigit(char c)
 
 bool IsHex(const std::string& str)
 {
+    if (str.size() == 0 || str.size()%2 != 0) return false;
```

Comment:
Please add spaces around the modulus operator.

Reply:
Before is like this